Resources/Prototypes/Tiles/floors.yml

```diff
@@ -1,5 +1,26 @@
+- type: tile
+  id: BaseStationTile
```

Comment:
while you're here, consider either in this PR or in a separate PR making common parents for tiles. There's a bunch of shared data that is copypasted a frankly insane amount of times.